components/centraldashboard/public/ajax_test_helper.js

```diff
@@ -26,3 +26,64 @@ export function mockRequest(component, response, respondWithError = false,
         component.addEventListener(responseEvent, () => resolve());
     });
 }
+
+/**
+ * Async sleep function which can pause execution flow or yield for next
+ * executions in the event-loop.
+ * @param {number} t Time in ms.
+ * @return {Promise<undefined>}
+ */
+export const sleep = (t) => new Promise((res) => setTimeout(res, t));
```

Comment:
For consistency's sake, use the `function` syntax rather than exporting a const. That's the preferred Google style.

Reply:
Done!